src/woo_publications/accounts/models.py

```diff
@@ -69,3 +71,29 @@ def get_full_name(self):
     def get_short_name(self):
         "Returns the short name for the user."
         return self.first_name
+
+
+class OrganisationMember(models.Model):
+    identifier = models.CharField(
+        _("identifier"),
+        help_text=_(
+            "The system identifier that uniquely identifies the user performing "
+            "the action."
+        ),
+        max_length=255,
+        unique=True,
+    )
+    naam = models.CharField(
+        _("naam"),
+        help_text=_("The display name of the user."),
+        max_length=255,
+    )
+
+    objects: ClassVar[OrganisationMemberManager] = OrganisationMemberManager()  # pyright: ignore[reportIncompatibleVariableOverride]
```

Comment:
@bart-maykin the ignore is fine here, I took a look and it's pyright being unnecessary pedantic IMO. I really hope these kind of things will be addressed with Ty from Astral...